### ISSUE TITLE
Regex parsing doesn't work when include tag has other attributes

### ISSUE THREAD

**Issue by pbcomm** [etylsarin/gulp-ssi#3]
Example NGINX include statement:

```
<!--# block name="empty" --><!--# endblock -->
<!--# include file="/dir/file.html" stub="empty" -->
```

Stub provides the default value when file is missing.

**Comment by etylsarin** [comment]
btw @rmoskal did your changes fixed this issue as well by a chance?

**Comment by rmoskal** [comment]
No.

**Comment by etylsarin** [comment]
fixed by v0.1.5
